### ISSUE TITLE
OpenVX CTS Failure - vxMapRemapPatch.MapRandomRemap

### ISSUE THREAD

**Issue by kiritigowda** [GPUOpen-ProfessionalCompute-Libraries/MIVisionX#558]
* HIP CPU Backend Failure

```
[ ======== ]
[ ALL DONE ] 5832 test(s) from 69 test case(s) ran
[ PASSED   ] 5831 test(s)
[ FAILED   ] 1 test(s), listed below:
[ FAILED   ] vxMapRemapPatch.MapRandomRemap
[ DISABLED ] 8190 test(s)

=================================
OpenVX Conformance report summary
=================================

To be conformant to the OpenVX baseline, 875 required test(s) must pass. 874 tests passed, 1 tests failed. FAILED.
To be conformant to the Vision conformance profile, 4957 required test(s) must pass. 4957 tests passed, 0 tests failed. PASSED.
Note: The 8190 disabled tests are optional and are not considered for conformance.
```

**Comment by kiritigowda** [comment]
* With Full Conformance
```
[ -------- ] tests from vxMapRemapPatch
[ RUN 0001 ] vxMapRemapPatch.MapRandomRemap ...

FAILED at /home/kiriti/develop/cts-failure/MIVisionX/conformance_tests/OpenVX-cts/test_conformance/test_remap.c:294
	Check failed for pixel (2, 0): 160

FAILED during execution of statement:
	/home/kiriti/develop/cts-failure/MIVisionX/conformance_tests/OpenVX-cts/test_conformance/test_remap.c:788:
		err_count += remap_check_pixel(input, output, x, y, coord_ptr->x, coord_ptr->y, interp_type, border, map, vx_true_e)

FAILED during execution of statement:
	/home/kiriti/develop/cts-failure/MIVisionX/conformance_tests/OpenVX-cts/test_conformance/test_remap.c:838:
		remap_validate_for_map(input, output, VX_INTERPOLATION_NEAREST_NEIGHBOR, border, map)

[ !FAILED! ] vxMapRemapPatch.MapRandomRemap
[ -------- ] 1 tests from test case vxMapRemapPatch
```
* With Filter
```
AGO_DEFAULT_TARGET=CPU LD_LIBRARY_PATH=./lib ./bin/vx_test_conformance --filter=vxMapRemapPatch.MapRandomRemap
VxTests version: 1.3
VCS version: 46d9196
Build config: Release

[ ======== ] Total 15289 tests from 69 test cases
Use test filter: vxMapRemapPatch.MapRandomRemap

Use global OpenVX context: FALSE


[ -------- ] tests from vxMapRemapPatch
[ RUN 0001 ] vxMapRemapPatch.MapRandomRemap ...
[     DONE ] vxMapRemapPatch.MapRandomRemap
[ -------- ] 1 tests from test case vxMapRemapPatch

[ ======== ]
[ ALL DONE ] 1 test(s) from 1 test case(s) ran
[ PASSED   ] 1 test(s)
[ FAILED   ] 0 test(s)
[ DISABLED ] 0 test(s)
```

**Comment by kiritigowda** [comment]
@LakshmiKumar23 this PR #592 fixes this issue, can you verify.

**Comment by kiritigowda** [comment]
Fix - #596 merged into TOT - need to unset env variable to close this issue - agoSetEnvironmentVariable("AGO_BUFFER_MERGE_FLAGS", "1")

**Comment by kiritigowda** [comment]
TOT Master - CPU CTS Results - Fixed as a regression issue

```
[ ======== ]
[ ALL DONE ] 5820 test(s) from 69 test case(s) ran
[ PASSED   ] 5820 test(s)
[ FAILED   ] 0 test(s)
[ DISABLED ] 8190 test(s)

=================================
OpenVX Conformance report summary
=================================

To be conformant to the OpenVX baseline, 863 required test(s) must pass. 863 tests passed, 0 tests failed. PASSED.
To be conformant to the Vision conformance profile, 4957 required test(s) must pass. 4957 tests passed, 0 tests failed. PASSED.
Note: The 8190 disabled tests are optional and are not considered for conformance.

#REPORT: 20210903011911 ALL 15277 8190 5820 5820 5820 0 (version 1.3)
```
